src/mono/System.Private.CoreLib/src/System/Reflection/Metadata/MetadataUpdater.cs

```diff
@@ -55,7 +55,8 @@ public static void ApplyUpdate(Assembly assembly, ReadOnlySpan<byte> metadataDel
 
         private static string InitializeApplyUpdateCapabilities()
         {
-            return ApplyUpdateEnabled(justComponentCheck: 1) != 0 ? "Baseline" : string.Empty ;
+            const string caps = "Baseline AddMethodToExistingType AddStaticFieldToExistingType NewTypeDefinition";
```

Comment:
N.B `NewTypeDefinition` is only partially supported.  Adding new instance fields doesn't work yet.  That's https://github.com/dotnet/runtime/issues/63643